content/c-sharp/concepts/math-functions/terms/cos/cos.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: '.Cos()'
+Description: 'Returns the cosine of a given angle.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Numbers'
+  - 'Arithmetic'
+  - 'Functions' 
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`Math.Cos()`** class method returns the cosine of a given angle.
+
+## Syntax
+
+```pseudo
+Math.Cos(angle);
+```
+
+The `Math.Cos()` method takes only one `double` type parameter - `angle` - an angle in radians. The `Math.Cos()` method returns the cosine of the `angle` as a `double` value or `NaN` (not a number) if the `angle` equals to:
+
+- `NaN`,
+- `NegativeInfinity`, or
+- `PositiveInfinity`.
+
+## Example
+
+The following example first converts `60` degrees to radians and then uses `Math.Cos()` method to return the cosine of that angle. Finally, the `Console.WriteLine()` function prints the result to the console:
+
+```cs
+using System;
+
+public class Example {
+  public static void Main(string[] args) {
+    double degrees = 60;
+    double radians = degrees * Math.PI/180;
+
+    double cosine = Math.Cos(radians);
+
+    Console.WriteLine("The cosine of " + degrees + " degrees is: " + cosine);
+  }
+}
+```
+
+The example will result in the following output:
+
+```shell
+The cosine of 60 degrees is: 0.5
```

Comment:
I get something like: .50000001. Perhaps round to one or two decimal places before writing to the console

Reply:
I added Math.Round(), but I still get 0.5. The cosine of 60 degrees should be 0.500000. 
![cos](https://user-images.githubusercontent.com/81328857/225278784-65abac30-0840-4f9d-b7df-b82586660e77.png)
Do you know what the issue could be?